dawn/test/integration-test/RegenerateIIRMain.cpp

```diff
@@ -27,11 +27,15 @@ int main(int argc, char* argv[]) {
   OptimizerContext optimizer(compiler.getDiagnostics(), optimizerOptions,
                              std::make_shared<dawn::SIR>());
 
-  auto copy_stencil = createCopyStencilIIRInMemory(optimizer);
-  UIDGenerator::getInstance()->reset();
-  IIRSerializer::serialize("reference_iir/copy_stencil.iir", copy_stencil, IIRSerializer::SK_Json);
-
-  auto lapl_stencil = createLapStencilIIRInMemory(optimizer);
-  UIDGenerator::getInstance()->reset();
-  IIRSerializer::serialize("reference_iir/lapl_stencil.iir", copy_stencil, IIRSerializer::SK_Json);
```

Comment:
here the wrong stencil was passed, therefore we added the scope to avoid such a problem in the future.